integration-cli/docker_cli_build_test.go

```diff
@@ -4583,17 +4583,21 @@ func TestBuildRenamedDockerfile(t *testing.T) {
 		t.Fatalf("Should have used dFile, output:%s", out)
 	}
 
-	out, _, err = dockerCmdInDir(t, ctx.Dir, "build", "--file=/etc/passwd", "-t", "test5", ".")
+	dirWithNoDockerfile, err := ioutil.TempDir(os.TempDir(), "empty")
+	if err != nil {
+		t.Fatal(err)
+	}
+	out, _, err = dockerCmdInDir(t, ctx.Dir, "build", fmt.Sprintf("--file=%s", dirWithNoDockerfile), "-t", "test5", ".")
```

Comment:
This one is supposed to point to an existing file so I'm not sure this is the same testcase.
We may need to create a temp dir+file and then point to that?